typing/env.ml

```diff
@@ -849,11 +833,11 @@ type lookup_error =
         container_class_type : string;
       }
   | Cannot_scrape_alias of Longident.t * Path.t
-  | Local_value_escaping of lock_item * Longident.t * escaping_context
-  | Once_value_used_in of lock_item * Longident.t * shared_context
-  | Value_used_in_closure of lock_item * Longident.t *
-      Mode.Value.Comonadic.error * closure_context
-  | Local_value_used_in_exclave of lock_item * Longident.t
+  | Local_value_escaping of Mode.Hint.lock_item * Longident.t * escaping_context
```

Comment:
Why do we keep `escaping_context` here?

Reply:
I suppose this will be fixed in a future PR.

Reply:
Yeah that's right - we will migrate those to the new mode hint system in the future.